tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -261,10 +263,11 @@ def test_harvesting_arxiv_workflow_manual_accepted(
     side_effect=fake_magpie_api_request,
 )
 def test_match_in_holdingpen_stops_pending_wf(
-    mocked_download_arxiv,
-    mocked_api_request_beard,
     mocked_api_request_magpie,
+    mocked_api_request_beard,
     mocked_package_download,
+    mocked_is_pdf_link,
+    mocked_download_arxiv,
```

Comment:
This one was using one less param as needed... it's the reverse case than the other one :), now it's ok.